app/javascript/app/components/accordion/accordion-component.jsx

```diff
@@ -8,75 +8,74 @@ import dropdownArrow from 'assets/icons/dropdown-arrow.svg';
 import layout from 'styles/layout.scss';
 import styles from './accordion-styles.scss';
 
-class Accordion extends PureComponent {
-  render() {
-    const {
-      className,
-      data,
-      handleOnClick,
-      openSlug,
-      children,
-      isChild,
-      hasNestedCollapse
-    } = this.props;
-    return (
-      <div className={className}>
-        {data &&
-          data.length > 0 &&
-          data.map((section, index) => {
-            let isOpen = index === 0;
-            if (openSlug && openSlug !== 'none') {
-              const isActiveInResults = data.some(d => d.slug === openSlug);
-              isOpen =
-                openSlug === section.slug ||
-                (index === 0 && !isActiveInResults);
-            } else {
-              isOpen = false;
-            }
-            const title = section.parent
-              ? `${section.parent.name} | ${section.title}`
-              : section.title;
-            return (
-              <section
-                key={`${section.slug}-${section.title}`}
-                className={styles.accordion}
+const Accordion = props => {
+  const {
+    className,
+    data,
+    handleOnClick,
+    children,
+    isChild,
+    hasNestedCollapse,
+    openSlug
+  } = props;
+
+  const getIsOpen = (section, index) => {
+    let isOpen = index === 0;
+    if (openSlug && openSlug !== 'none') {
+      const isActiveInResults = data.some(d => d.slug === openSlug);
+      isOpen = openSlug === section.slug || (index === 0 && !isActiveInResults);
+    } else {
+      isOpen = false;
+    }
+    return isOpen;
+  };
+
+  return (
+    <div className={className}>
+      {data &&
+        data.length > 0 &&
+        data.map((section, index) => {
+          const isOpen = getIsOpen(section, index);
+          const title = section.parent
+            ? `${section.parent.name} | ${section.title}`
```

Comment:
I see that it was in the code before, but why we're using here bitwise `OR` instead of the logical one? Apparently it's not a bitwise operation, so I'd like to ask you for changing this small bit because it can cause an error one day :pray:
```suggestion
            ? `${section.parent.name} || ${section.title}`
```